frontend/src/utils/address.ts

```diff
@@ -131,6 +131,18 @@ export const isAddressSafe = async (name: string, userAddress: string, stealthAd
   // Check if address has contributed to Gitcoin Grants
   // TODO
 
+  // Check if address is an ERC20
+  const desIsERC20 = await isERC20(destinationAddress, provider);
+  if (desIsERC20) {
+    reasons.push(`${i18n.tc('Utils.Address.it')} ${i18n.tc('Utils.Address.is')} ${i18n.tc('Utils.Address.is-erc20')}`);
+  }
+
+  // Check if address is an ERC721
+  const desIsERC721 = await isERC721(destinationAddress, provider);
+  if (!desIsERC20 && desIsERC721) {
+    reasons.push(`${i18n.tc('Utils.Address.it')} ${i18n.tc('Utils.Address.is')} ${i18n.tc('Utils.Address.is-erc721')}`);
```

Comment:
Should we just push one `'Utils.Address.is-erc721'` string instead of composing? Seems that would scale better

Reply:
I am changing to a single string, but this construction matches the way existing error messages are constructed for withdrawl

Reply:
Ah, ok yea we might need to do some refactoring then if we ever support more languages, but thanks!